sale_order_line_input/views/sale_order_line_view.xml

```diff
@@ -0,0 +1,239 @@
+<?xml version="1.0" ?>
+<!-- Copyright 2018 Tecnativa - Carlos Dauden
+     Copyright 2023 Tecnativa - Carolina Fernandez
+     Copyright 2024 Tecnativa - Juan José Seguí
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_sales_order_line_input_tree" model="ir.ui.view">
+        <field name="name">sale.order.line.input.list.view</field>
+        <field name="model">sale.order.line</field>
+        <field name="priority">20</field>
+        <field name="arch" type="xml">
+            <list
+                decoration-info="invoice_status == 'to invoice'"
+                decoration-warning="state == 'draft'"
+                decoration-muted="state == 'cancel'"
+                decoration-success="invoice_status == 'invoiced'"
+                editable="top"
+                create="true"
+            >
+                <!-- We do not display the type because we don't want the user to be bothered with
+                that information if he has no section or note. -->
+                <field
+                    name="force_company_id"
+                    optional="hide"
+                    readonly="order_id"
+                    options="{'no_create': True}"
+                />
+                <field
+                    name="order_id"
+                    string="Order"
+                    required="0"
+                    placeholder="New"
+                    readonly="order_id"
+                    domain="[('state', '!=', 'cancel'),('company_id', '=', company_id)]"
+                    force_save="1"
+                    options='{"no_create": True}'
+                />
+                <field
+                    name="order_partner_id"
+                    context="{'res_partner_search_mode': 'customer', 'default_is_company': True, 'default_customer_rank': 1}"
+                    options='{"always_reload": True}'
+                    required="1"
+                    readonly="order_partner_id and order_id"
+                />
+                <field name="write_date" widget="date" optional="hide" />
+                <field
+                    name="product_id"
+                    readonly="not product_updatable"
+                    required="not display_type"
```

Comment:
This is something I have to comment on every migration...
The objective of this module is to manage the order lines from a tree view, but keeping as much as possible the behaviour they have in the normal order view.
To do this, what you have to do in the migration of this module is to go to compare the order view and put the same fields with the same attributes (except the conditions in which ‘parent.’ is used).

This is just an example of the attributes that have not been checked, but there are others like with, string, etc. that are not seen here either:
https://github.com/odoo/odoo/blob/c84b67e5dca1a74e1a5862e754455944015f4105/addons/sale/views/sale_order_views.xml#L531